docs/rule_template.vm

```diff
@@ -0,0 +1,31 @@
+# Attributes
+
+Name           | Type                               | Default            | Description
+---------------|------------------------------------|--------------------|------------
+#foreach ($attr in $ruleInfo.getAttributeList())
```

Comment:
😰

Reply:
Yeah, I know. A very weird choice for a templating language but oh well, that’s enterprise for you.